src/resources/postcss/tickets-admin-settings.pcss

```diff
@@ -14,18 +14,16 @@
 	p.event-tickets__admin-banner-help-text {
 		max-width: 80%;
 	}
+}
 
+.event-tickets__admin-banner-help-links-wrapper {
```

Comment:
Should this be within the `.event-tickets__admin-banner` for structure or does that not matter?

Reply:
I moved them out as @juanfra suggested in his CR. Not much sure how BEM in PostCSS works.

Reply:
@sc0ttkclark @rafsuntaskin Ideally we should have them outside of it. We're following the same rules we have for the rest of the CSS using BEM, where each block has to be defined independently (with the exception of the wrappers we have, like `tribe-common`). There's more info about this here: https://github.com/the-events-calendar/the-events-calendar/tree/master/src/resources/postcss

And you can check this stylesheet as a reference: https://github.com/the-events-calendar/the-events-calendar/blob/master/src/resources/postcss/views/full/month/_calendar-event.pcss